cli/azd/cmd/root.go

```diff
@@ -320,6 +321,11 @@ func NewRootCmd(
 			},
 		}).
 		UseMiddleware("hooks", middleware.NewHooksMiddleware)
+	root.
+		Add("add", &actions.ActionDescriptorOptions{
```

Comment:
Should we condition adding this command based on enabling the alpha feature?   Like, show the command in the list only when the feature is enabled, otherwise we don't show it?

Reply:
@vhvb1989

I believe based on `azd help` is currently working, which is based on the work you had previously done to separate out our help from `cobra.cmd`, our root help [menu](https://github.com/Azure/azure-dev/blob/main/cli/azd/cmd/testdata/TestUsage-azd.snap) doesn't automatically light up new commands. It actually requires explicit configuration within the codebase. You may notice that TestUsage-azd.snap is unchanged in this PR.

Correct if I'm mistaken, but the UX concern about "listing the command only when it is enabled" isn't applicable here, since it is never listed. I think this is acceptable for alpha.

On the overall UX thinking, I think it's favorable to have `azd <alpha-command> --help` still show up, and I think it's favorable to have `azd <alpha-command>` provide a follow-up on what to do next. This is much preferable to having "invisible commands". As always, I'm happy to take an issue or a design proposal for discussion separately.

Reply:
Captured the line item for alpha advancement to surface in root menu in #4538